.gitignore

```diff
@@ -15,6 +15,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+.DS_Store
```

Comment:
I code on Mac, and it adds this file basically everywhere - I think it would be a good idea to include this in the gitignore in case anyone else codes on Mac for this project.

Reply:
I also code on Mac, .ds_store is a desktop stores file and should be deleted and not added to a git ignore